Turn/src/Game.cpp

```diff
@@ -203,7 +203,14 @@ void Game::Intermission(){
     for (int choice=0; IsPlaying;){
         ClearScreen();
         cout << "*--------- Intermission ----------* " << endl << endl;
-		_Player->DisplayInventory();
+		
+	int tempLevel = _Player->GetLevel();
+	if (_Level != tempLevel) {
+		_Level = tempLevel;
+		cout << "Congratulation on leveling up!" << endl;
+	}
```

Comment:
Hmm, on second thought-- this level up message would look better placed after winning a battle (after showing the items you won), if you're up for it.

Reply:
I believe it already does this inside Player::AddExperience